### ISSUE TITLE
Unused settings for CKEditor

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#3406]
Currently the following settings are set up for the CKEditor in the methods `SetRichTextParameters()`and `CustomerSetRichTextParameters()` in _Kernel/Output/HTML/Layout.pm_. Some of these are unused:

- Height
- Width
- Lang.SplitQuote, unused
- Lang.RemoveQuote, unused
- Plugins
- Toolbar
- ToolbarWithoutImage
- ToolbarMidi, unused, commented out
- ToolbarMini, unused, commented out
- PictureUploadAction

It should be checked whether the unused setting are really not needed. If not needed then they can be removed.

**Comment by stefanhaerter** [comment]
Perhaps it does also make sense to have the CodeMirror editor in mind when checking. To the best of my knowledge, CodeMirror currently affects AdminTicketMask, AdminDynamicFieldSet, AdminITSMConfigItem and XSLT Mappings im Webservices.

**Comment by bschmalhofer** [comment]
Yes, some settings might also apply to CodeMirror. At least Height and Width should also apply to CodeMirror, unless these settings are ditched altogether.

**Comment by stefanhaerter** [comment]
As specified here

https://github.com/RotherOSS/otobo/blob/823daad743986c063d1195f9054ba14f32f99e71/var/httpd/htdocs/skins/Agent/default/css/Core.CodeMirrorEditor.css#L31-L36

I set the height of CodeMirror editors css-wise to 10 lines plus padding. Maybe it would be a nice enhancement to offer a sysconfig setting for setting the number of lines which is then translated into pixels.

**Comment by chzauleck** [comment]
I would suggest handling the CodeMirror side of things in a seperate issue, as the code side of it is completely unrelated to ckeditor at the moment?

**Comment by stefanhaerter** [comment]
I suggest to move this to milestone 11.0. What is your opinion, @bschmalhofer and @chzauleck?

**Comment by bschmalhofer** [comment]
Or move it to the the wishlist, and the triage project, It is just some strangeness in the code.

**Comment by stefanhaerter** [comment]
Adhering to @bschmalhofer 's comment and moving it to the wishlist.
